linux_pipeline/Jenkinsfile_lis_rpm_hyperv

```diff
@@ -119,6 +122,31 @@ stage ("Deploy stage") {
     }
 }
 
+stage('Local Hyper-V performance') {
```

Comment:
declarative pipeline sintax won't worn in this case.
you can use somthing like: 
stage (){
    node(){
         withcreds (){
              powershell
         }
    }
}